torch/jit/_state.py

```diff
@@ -117,3 +117,6 @@ def _set_jit_function_cache(key, value):
     # only free functions currently supported
     assert isinstance(value, torch.jit.ScriptFunction)
     _jit_caching_layer[key] = value.qualified_name
+
```

Comment:
I noticed that in commit history of this PR, there is one titled as "Merge branch 'master' of https://github.com/pytorch/pytorch into cach… 

@nikithamalgifb
nikithamalgifb committed 4 days ago
"

It looks like you accidentally merged master into this branch in order to update this branch to HEAD. This will cause trouble for future commits if you add any because it treats code changes from other people as made by yourself, making future merge conflicts harder to resolve. The correct approach is to do something like this:
```
git checkout master
git pull origin master
git checkout cached_functions
git rebase master
```